### ISSUE TITLE
LocalDensity.compute does not accept keyword args

### ISSUE THREAD

**Issue by bdice** [glotzerlab/freud#138]
**[Original report](https://bitbucket.org/glotzer/freud/issue/138) by Joshua Anderson (Bitbucket: [joaander](https://bitbucket.org/joaander), GitHub: [joaander](https://github.com/joaander)).**

----------------------------------------

Code:
```
counter = freud.density.LocalDensity(r_cut=rcut, volume=1, diameter=1)
counter.compute(box=box, ref_points=pos, points=pos)
```

Error:
```
TypeError: compute() got an unexpected keyword argument 'box'
```

Based on the documentation, I would expect it to take in keyword args.

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

Addresses Issue #138

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

Addresses Issue #138

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

Merged in Issue138 (pull request #93)

Addresses Issue #138

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

Addresses Issue #138

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

Merged in Issue138 (pull request #93)

Addresses Issue #138

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

Addresses Issue #138

**Comment by bdice** [comment]
**Original comment by Joshua Anderson (Bitbucket: [joaander](https://bitbucket.org/joaander), GitHub: [joaander](https://github.com/joaander)).**

----------------------------------------

I haven't tested it, but it appears that pull request #93 resolves this issue.

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

resolved in pull request #93
